lib/helpers/utils.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Do we _need_ to do a `.map .join` for this method?
there appear to be a _few_ other ways of doing this: http://stackoverflow.com/questions/1026069/how-do-i-make-the-first-letter-of-a-string-uppercase-in-javascript including a CSS-only solution... 😉